src/sentry/models/organizationonboardingtask.py

```diff
@@ -39,6 +39,7 @@ class OnboardingTask:
 
 class OnboardingTaskStatus:
     COMPLETE = 1
+    # deprecated - no longer used
     PENDING = 2
```

Comment:
I meant also commenting out the value, so that it can't be used, but it is still visible in the future :) 
```suggestion
    # deprecated - no longer used
    # PENDING = 2
```